src/ui/public/chrome/directives/loading_indicator/loading_indicator.less

```diff
@@ -0,0 +1,50 @@
+@import "~ui/styles/variables";
+
+@loadingIndicatorBackgroundSize: 400px;
+@loadingIndicatorHeight: 2px;
+@loadingIndcatorColor1: @kibanaPink1;
+@loadingIndcatorColor2: @kibanaPink2;
+
+/**
+ * 1. Child is absolutely positioned.
+ * 2. Make sure indicator isn't wider than the screen.
+ */
+.loadingIndicator {
+  position: relative; // 1
+  max-width: 100%; // 2
+  overflow: hidden; // 2
+  height: @loadingIndicatorHeight;
+
+  &.ng-hide {
+    visibility: hidden;
+    opacity: 0;
+    transition-delay: 0.25s;
+  }
+}
+
+  .loadingIndicator__bar {
```

Comment:
Do we have a guideline for how to name css classes?  (or maybe airbnb does...).  I see myVariable, my-variable and my__variable.  Might be good to pick one and stick with it, unless there is another reason for it.